index.qmd

```diff
@@ -42,13 +42,13 @@ The diagram below depicts how some of the cloud-optimized formats discussed in t
 
 Notes:
 
-* Some data formats cover multiple data types, specifically:
-    * GeoJSON can be used for both vector and point data.
-    * HDF5 can be used for point data or data cubes (or both via groups).
-    * GeoParquet and FlatGeobuf can be used for vector data or point data.
-* LAS files are intended for 3D points, not 2D points (since COPC files are compressed LAS files, same goes for COPC files).
-* [TopoJSON](https://github.com/topojson/topojson) (an extension of GeoJSON that encodes topology) and [newline-delimited GeoJSON](https://stevage.github.io/ndgeojson/) are types of GeoJSON worth mentioning but not explicitly represented in the diagram.
-* GeoTIFF and GeoParquet are geospatial versions of the non-geospatial file formats TIFF and Parquet, respectively. FlatGeobuf builds upon the non-geospatial [flatbuffers](https://github.com/google/flatbuffers) serialization library (though flatbuffers is not a standalone file format)
+- Some data formats cover multiple data types, specifically:
+    - GeoJSON can be used for both vector and point data.
```

Comment:
This is confusing, vector is points, lines, and polygons

Reply:
I used the exact wording from the last revision; I just changed the '*' list to a '-' list accidentally with my editor. 

Perhaps it is meant to mean point clouds? I have not seen Geojson used for point clouds at scale.

Reply:
@wildintellect give this comment another look with the new changes, but we may want to check in with @abarciauskas-bgse and @kylebarron .

Reply:
I agree that I've never seen GeoJSON used for point data.

Reply:
Not a blocker for this PR